priya/javascript/Assignment17/JsAssign17Q1.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Assignment17Q1</title>
+  </head>
+  <body>
+    Name: <input type="text"id="name"placeholder="Enter Name"><br>
+    <br>
+    Roll no:<input type="text"id="rollno" placeholder="Enter roll number "><br>
+    <br>
+    <label for="marks"><b>Marks :</b></label><br>
+    <br>
+    Maths :<input type="text" id="math" placeholder="Enter marks of math"><br>
+    <br>
+    Science :<input type="text" id="Science"placeholder="Enter marks of Science"><br>
+    <br>
+    History :<input type="text"id="History" placeholder="Enter marks of History"><br>
+    <br>
+    <br>
+    <button onclick="submit()">Submit</button>
+    <p id="demo"></p>
+    <script>
+      function submit() {
+        var roll_number, name, math, Science, history, total, avg;
+        name = document.getElementById("name").value;
+        var upper_name = name.toUpperCase();
+        roll_number = document.getElementById("rollno").value;
+        math = parseInt(document.getElementById("math").value);
+        Science = parseInt(document.getElementById("Science").value);
+        history = parseInt(document.getElementById("History").value);
+        if (isNaN(math)) {
+          alert("Marks cant be alpha numeric");
+          math = parseInt(prompt("enter marks again"));
```

Comment:
don't use prompt..instead place the cursor in the text field where the marks are alphanumeric and try to use exception handling